static/src/javascripts/projects/commercial/modules/consentless/define-slot.ts

```diff
@@ -1,16 +1,29 @@
 import { renderAdvertLabel } from '../dfp/render-advert-label';
 
-const getOptOutSlotName = (dfpSlotName: string): string => {
-	if (dfpSlotName.includes('top-above-nav')) {
-		return 'homepage-lead';
+const isArticle = window.guardian.config.page.contentType === 'Article';
+const isFront = window.guardian.config.page.isFront;
+
+const getOptOutSlotName = (slotName: string, inlineId?: number): string => {
+	// front, liveblog, article inlines all have slightly different mappings, so we prefix them
+	if (slotName === 'inline') {
+		if (isArticle) {
+			// article inlines are suffixed with 1 for inline1 and 2-plus for all other inlines as they have different size mappings
+			return `article-inline${inlineId === 1 ? '1' : '2-plus'}`;
+		}
+		return `${isFront ? 'front-' : 'liveblog-'}${slotName}`;
```

Comment:
Is it true that any inline slot that is not an Article or a Front, must be a liveblog?

Reply:
Ah you're right🤦‍♂️, there are some others like galleries with inline slots, perhaps fallback to just an 'inline' adSlot name?